major_event_log/feeds.py

```diff
@@ -18,85 +65,55 @@ class MajorEventLogFeed(Atom1Feed):
 
     mime_type = 'application/xml'
 
-    def add_root_elements(self, handler):
-        handler.addQuickElement(u'title', self.feed['title'])
-        handler.addQuickElement(
-            u'link', '', {u'rel': u'self', u'href': self.feed['feed_url']}
-        )
-        handler.addQuickElement(u'id', self.feed['id'])
-        handler.startElement(u'author', {})
-        handler.addQuickElement(u'name', self.feed['author_name'])
-        handler.addQuickElement(u'uri', self.feed['author_link'])
-        handler.endElement(u'author')
-        handler.addQuickElement(u'subtitle', self.feed['subtitle'])
-        handler.addQuickElement(
-            u'link',
-            '',
-            {u'rel': u'first', u'href': '{}?p=1'.format(self.feed['link'])}
-        )
-        handler.addQuickElement(
-            u'link',
-            '',
-            {
-                u'rel': u'last',
-                u'href': '{}?p={}'.format(self.feed['link'],
-                                          self.feed['last_link'])
-            }
-        )
-        if self.feed.get('prev_link', None) is not None:
-            handler.addQuickElement(
-                u'link',
-                '',
-                {
-                    u'rel': u'previous',
-                    u'href': '{}?p={}'.format(self.feed['link'],
-                                              self.feed['prev_link'])
-                }
-            )
-        if self.feed.get('next_link', None) is not None:
-            handler.addQuickElement(
-                u'link',
-                '',
-                {
-                    u'rel': u'next',
-                    u'href': '{}?p={}'.format(self.feed['link'],
-                                              self.feed['next_link'])
-                }
-            )
-
-
-class LatestEventsFeed(Feed):
+
+class PaginatedFeedMixin(object):
+    paginator = None
+    page = 1
+    items_per_page = 2
```

Comment:
Absolutely. 

I was testing this out and didn't want to add 10+ events in the DB